social-network/src/Box2.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import mArticles from './missed-articles.json';
+
+console.log(mArticles);
+
+class Box2 extends React.Component {
+    render() {
+        return (
+            <div>
+                {mArticles.map((item, index)=>{
+                 return <div className="mBoxes" key={index}>
```

Comment:
See comment below, make this a component so it can be reused for both missed articles and your articles. Don't forget proptypes.

Reply:
How do you add different CSS styles if you use the same component for the two views?  Or should i not worry about that right now.  haha

Reply:
You can pass in a prop to do different styling based on the prop value, or give it a different `className`.